core-bundle/contao/library/Contao/ArrayUtil.php

```diff
@@ -149,4 +149,46 @@ public static function mapRecursive(callable $fn, array $arr): array
 	{
 		return array_map(static fn ($item) => \is_array($item) ? self::mapRecursive($fn, $item) : $fn($item), $arr);
 	}
+
+	/**
+	 * Add, remove or replace values from the current array based on your configuration.
+	 */
+	public static function filterValuesToIgnore(array $current, array $filter, string|null $default = null): array
```

Comment:
```suggestion
	/**
	 * Add, remove or replace values from the list based on your configuration.
	 */
	public static function alterListByConfig(array $list, array $config): array
```
Just a suggestion, WDYT @leofeyer ?

Reply:
See 4a7201edead041bb70ceedbca8e6d60afdbd3f25